app/views/layouts/angular.html.haml

```diff
@@ -14,7 +25,7 @@
     %start_menu
 
 
-    - if Rails.env.production?
+    - if Rails.env.production? and ENV['ERRBIT_HOST']
```

Comment:
Not sure I understand why we're hiding the turn off angular option if Errbit isn't set up?